src/components/topbar/TopbarBadge.test.ts

```
@@ -1,5 +1,5 @@
-import { mount } from '@vue/test-utils'
-import Popover from 'primevue/popover'
+import { render, screen } from '@testing-library/vue'
```

ESLint

[error] Unable to resolve path to module '@testing-library/vue'.

(import-x/no-unresolved)